Update enrolling-mtrp-managed-service.md (#10031)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/rooms/enrolling-mtrp-managed-service.md

# Accessing the Pro Management portal

> In order to access the Pro portal you will first need to have Microsoft Teams Rooms Pro licenses added to your tenant. The first time the [Teams Rooms Pro Management portal](https://portal.rooms.microsoft.com/) is accessed must be done by a tenant global administrator in order to consent the application. 

Reviewer: ```suggestion
To access the Pro Management portal you need to have Microsoft Teams Rooms Pro licenses added to your organization. The first time the [Teams Rooms Pro Management portal](https://portal.rooms.microsoft.com/) is accessed, it should be done by a global administrator from your organization in order to consent to the application. 
```

> The global admin can then [assign roles](https://learn.microsoft.com/en-us/microsoftteams/rooms/rooms-pro-rbac) to other users in the tenant for accessing the portal. The Built-in "Teams Rooms Pro Manager" role will give users full control over the Pro portal so it can be managed without a tenant global admin account. Tenant global admins will always have "Teams Rooms Pro Manager" permissions to the Pro portal, if roles are ever accidentally removed or if users are deleted you can always use a global admin account to get back to the role assignments. 

Reviewer: ```suggestion
The global admin can then [assign roles](/microsoftteams/rooms/rooms-pro-rbac) to other users in the organization to access the portal. The built-in Teams Rooms Pro Manager role will give users full control over the Pro Management portal so it can be managed without a tenant global admin account. Tenant global admins will always have Teams Rooms Pro Manager permissions to the Pro Management portal. If roles are ever accidentally removed or if users are deleted you can always use a global admin account to restore the role assignments. 
```

## Assign users to the Teams Rooms Pro Manager role

Complete the following steps to assign users to the Teams Rooms Pro Manager role:

1. Log in to the [Teams Rooms Pro Management portal](https://portal.rooms.microsoft.com/) with the tenant global admin account.
2. Navigate to **Settings** > **Settings** > **Roles** and then select **Teams Rooms Pro Manager**.
3. Under **Teams Rooms Pro Manager**, select the **Assignments** tab and then select **Add Assignment**.
4. Follow the wizard to name the assignment and select the users who should be added to it. The assignment will apply to all rooms and room groups.
5. At the end of the assignment wizard, select **Add assignment**.

> Users who are assigned the Teams Rooms Pro Manager role have full access to all settings and rooms in the Pro portal. You can also assign users "Site Lead" or "Site Technician" default roles or define customer roles for those responsible for the day-to-day management and monitoring of Teams Rooms. 

Reviewer: ```suggestion
Users who are assigned the Teams Rooms Pro Manager role have full access to all settings and rooms in the Pro Management portal. They can also assign users to the Site Lead or Site Technician default roles or define customer roles for those responsible for the day-to-day management and monitoring of Teams Rooms. 
```
